core/src/executor/evaluate/function.rs

```diff
@@ -1,6 +1,6 @@
 use {
     super::{ChronoFormatError, EvaluateError, Evaluated},
-    crate::{ast::TrimWhereField, data::Value, result::Result},
+    crate::{ast::TrimWhereField, data::Value, prelude::DataType, result::Result},
```

Comment:
`prelude` is for crate users so it would be better to use explicit module imports.
Here, we can simply use `crate::ast::DataType`